### PR TITLE
Dp 50 as a user i would like to have an overview of my bookings

### PR DESCRIPTION
**Pull request by sankeets**

### CODE REVIEW

File: backend/django_backend/bookings/tests.py [hunk 1]
```diff
@@ -56,3 +59,33 @@ def test_get_available_timeslots(self):
...
+            booking_date="2024-05-10",
+            start_time="10:00",
+            end_time="12:00"
+        )
+        
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.user)
+
+    def test_cancel_booking_sends_email(self):
+        response = self.client.delete(reverse('booking-delete', kwargs={'booking_id': self.booking.id}))
```
Comment (VegSja) [inline backend/django_backend/bookings/tests.py:86]: This won't work after it was mapped to a POST request

Comment (VegSja) [reply backend/django_backend/bookings/tests.py:86]: I think, not sure

File: frontend/dogplus-frontend/src/components/serviceProvider/BookingsOverview.tsx [hunk 2]
```diff
@@ -139,6 +187,8 @@ export const BookingsOverview = () => {
         No data available.
       </div>
     );
+
+  console.log(bookingsForSelectedDate);
```
Comment (VegSja) [inline frontend/dogplus-frontend/src/components/serviceProvider/BookingsOverview.tsx:191]: Should not be here

**Review by VegSja — CHANGES_REQUESTED** [review]

**Comment by VegSja** [comment]
Otherwise it looks good

**Comment by VegSja** [comment]
Should also have some sort of header here that says "Your bookings" or something like this
Screen Shot 2024-05-19 at 16 42 36

File: backend/django_backend/authentication/migrations/0004_auto_20240502_1649.py
Comment (VegSja) [inline backend/django_backend/authentication/migrations/0004_auto_20240502_1649.py:1]: The changes to this file makes it impossible to login as one of the test users

**Review by VegSja — CHANGES_REQUESTED** [review]

File: backend/django_backend/bookings/tests.py (hunk 1 above)
Comment (sankeets) [reply backend/django_backend/bookings/tests.py:86]: Could you reproduce test results, send screen pls

Comment (sankeets) [reply backend/django_backend/bookings/tests.py:86]: Tenker uansett vi kan approve denne, siden epost funksjonaliteten bare ligger som et konsept som kan utvides og ikke faktisk fungerende akk nå

**Comment by sankeets** [comment]
> Should also have some sort of header here that says "Your bookings" or something like this Screen Shot 2024-05-19 at 16 42 36

Added tabs to differentiate between the socials and bookings

**Review by VegSja — APPROVED** [review]
LGTM :rocket:

**Pull request merged**